apps/taxes/src/modules/taxes/providers/taxjar/taxjar-calculate.ts

```diff
@@ -5,14 +5,24 @@ import {
   TaxBaseLineFragment,
   TaxDiscountFragment,
 } from "../../../../../generated/graphql";
-import { FetchTaxesLinePayload, ResponseTaxPayload } from "../../types";
 import { ChannelConfig } from "../../../channels-configuration/channels-config";
 import { taxLineResolver } from "../../tax-line-resolver";
+import { ResponseTaxPayload } from "../../types";
 
 const formatCalculatedAmount = (amount: number) => {
   return Number(amount.toFixed(2));
 };
 
+type FetchTaxesLinePayload = {
+  id: string;
+  quantity: number;
+  taxCode?: string | null;
+  discount: number;
+  chargeTaxes: boolean;
+  unitAmount: number;
+  totalAmount: number;
+};
```

Comment:
where does it come from? Is it your internal typo or something that comes from saleor or taxjar?